.github/workflows/ci.yaml

```diff
@@ -44,3 +44,39 @@ jobs:
               uses: "docker://oskarstark/phpstan-ga"
               with:
                   args: analyse
+
+    auto-merge:
+        name: "Auto-Merge"
+
+        runs-on: "ubuntu-latest"
+
+        needs:
+            - "php-cs-fixer"
+            - "composer-normalize"
+            - "phpstan"
```

Comment:
Since the order does not matter, perhaps it will be easier to maintain if these job names are sorted?

See https://plugins.jetbrains.com/plugin/5919-lines-sorter!

🤓